Create ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md (#2368)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-18-3-translation-added-from-public-options-will-now-have-higher-priority-than.md

---
date: ""

> title: Public option translations now have higher priority than plugin po files

Reviewer: ```suggestion
title: Public option translations now have higher priority than translations from plugin PO files
```

> title: Translation added from public options will now have higher priority than plugin po files (#6932)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Public option translations now have higher priority than plugin po files
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60268
version: 1020.18.3
---

> Translation added from public options will now have higher priority than plugin po files (#6932)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, translations from plugin po files took precedence over translations added via public options, which could lead to inconsistencies. With this change, translations added via public options will now have a higher priority than translations from plugin po files. This ensures that custom translations specified in the public options are always used, providing a more consistent and predictable translation behavior for users. Existing translations in plugin po files that conflict with public option translations will be overridden.
```

> Previously, translations from plugin po files took precedence over translations added via public options, which could lead to inconsistencies. With this change, translations added via public options will now have a higher priority than translations from plugin po files. This ensures that custom translations specified in the public options are always used, providing a more consistent and predictable translation behavior for users. Existing translations in plugin po files that conflict with public option translations will be overridden.

Reviewer: ```suggestion
Previously, translations from plugin PO files (file format used for UI translations) took precedence over translations added via public options, which could lead to inconsistencies. With this change, translations added via public options will now have a higher priority than translations from plugin PO files. Existing translations in plugin PO files that conflict with public option translations will be overridden. This ensures a consistent and predictable translation behavior. 
```
